src/ebay-menu/types.ts

```diff
@@ -1,2 +1,23 @@
+import { ComponentProps } from 'react'
+
 export type EbayMenuType = 'radio' | 'checkbox'
 export type EbayMenuPriority = 'primary' | 'secondary' | 'none'
+
+type ContainerDivProps = Omit<ComponentProps<'div'>, 'onKeyDown' | 'onChange' | 'onSelect'>
+type ContainerSpanProps = Omit<ComponentProps<'span'>, 'onKeyDown' | 'onChange' | 'onSelect'>
+
+export type SelectCallback = (i: number, checked: boolean) => void
+export type ChangeCallback = (i: number, checked: boolean) => void
+export type KeyDownCallback = (i: number, checked: boolean) => void
```

Comment:
@HenriqueLimas i see main branch also has support for checkedindexes array, its better to move the API to that. what do u think?